src/test/java/com/ifactory/press/db/solr/spelling/suggest/MultiSuggesterTest.java

```diff
@@ -138,17 +158,18 @@ private void assertNoSuggestions() throws SolrServerException {
     }
 
     private void assertSuggestions() throws SolrServerException {
-        Suggestion suggestion = assertSuggestionCount("t", 8, "all");
+        Suggestion suggestion = assertSuggestionCount("t1", 5, "all");
         // TITLE occurs once in a high-weighted field; t1-t4, etc each occur twice, t5 once, their/time occur once
         // 'the' and 'to' occur too many times and get excluded
-        assertEquals(TITLE, suggestion.getAlternatives().get(0));
-        for (int i = 1; i <= 5; i++) {
+        // assertEquals(TITLE, suggestion.getAlternatives().get(0));

```

Comment:
So this line is saying hey the first suggestion should be for the full title. I think the alternatives are ranked by likihood or something. We should probably see if this passes

Reply:
It doesn't because of the diversion into  t1(+num)  ie t11..t15 instead of t to t1 or 'the'

Reply:
I put one to match on 169